site/_posts/2017-05-08-doing-just-fine.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Last Week in ManageIQ: Doing just Fine"
+author: skateman
+date: 2017-05-09
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi! As you know? We're all doing just *Fine* (pun intended) and I would like to discuss some of the details. So let's get started.
```

Comment:
I think @jntullo is asking `should this be "As you know"?` 

i.e. replacing `now` with `know` , but not to add the question mark which is outside of the quotes